CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)
 link_directories(${CMAKE_BINARY_DIR}/bin)
 set(EXECUTABLE_OUTPUT_PATH ${CMAKE_BINARY_DIR}/bin)
 
-add_executable(lr-example src/regression.cpp)
+add_executable(lr-example src/main.cpp)
```

Comment:
We are testing everything temporarily on `main.cpp` file. CSV files will be tested later.